scaleway/testdata/object-bucket-destroy-force.cassette.yaml

```diff
@@ -0,0 +1,568 @@
+    code: 404
+    duration: ""
+- request:
+    body: ""
+    form: {}
+    headers:
+      Authorization:
+      - AWS4-HMAC-SHA256 Credential=SCWK721NRK9BFRHW1CS7/20220214/fr-par/s3/aws4_request,
+        SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=eaf99afe64337514725f76db808fd3fb82217e4e16aae65c17f6811a89995c1a
+      User-Agent:
+      - aws-sdk-go/1.42.48 (go1.17.6; linux; amd64)
+      X-Amz-Content-Sha256:
+      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      X-Amz-Date:
+      - 20220214T140855Z
+    url: https://test-acc-scaleway-object-bucket-force-7443382734974225702.s3.fr-par.scw.cloud/?cors=
+    method: GET
+  response:
+    body: |-
+      <?xml version='1.0' encoding='UTF-8'?>
+      <Error><Code>NoSuchCORSConfiguration</Code><Message>The CORS configuration does not exist</Message><RequestId>tx119d349c99aa482c9a1da-00620a6277</RequestId></Error>
+    headers:
+      Content-Type:
+      - application/xml
+      Date:
+      - Mon, 14 Feb 2022 14:08:55 GMT
+      X-Amz-Id-2:
+      - tx119d349c99aa482c9a1da-00620a6277
+      X-Amz-Request-Id:
+      - tx119d349c99aa482c9a1da-00620a6277
+    status: 404 Not Found
+    code: 404
+    duration: ""
+- request:
+    body: ""
+    form: {}
+    headers:
+      Authorization:
+      - AWS4-HMAC-SHA256 Credential=SCWK721NRK9BFRHW1CS7/20220214/fr-par/s3/aws4_request,
+        SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=c7672865c45a4c177d63f9635c34fb1d283edc56ba139ceb24be8b945ae9e1e5
+      User-Agent:
+      - aws-sdk-go/1.42.48 (go1.17.6; linux; amd64)
+      X-Amz-Content-Sha256:
+      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      X-Amz-Date:
+      - 20220214T140855Z
+    url: https://test-acc-scaleway-object-bucket-force-7443382734974225702.s3.fr-par.scw.cloud/?versioning=
+    method: GET
+  response:
+    body: |-
+      <?xml version='1.0' encoding='UTF-8'?>
+      <VersioningConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"/>
+    headers:
+      Content-Length:
+      - "113"
+      Content-Type:
+      - text/plain
+      Date:
+      - Mon, 14 Feb 2022 14:08:56 GMT
+      X-Amz-Id-2:
+      - txd46aba3bc1ea4420a7a5f-00620a6277
+      X-Amz-Request-Id:
+      - txd46aba3bc1ea4420a7a5f-00620a6277
+    status: 200 OK
+    code: 200
+    duration: ""
+- request:
+    body: ""
+    form: {}
+    headers:
+      Authorization:
+      - AWS4-HMAC-SHA256 Credential=SCWK721NRK9BFRHW1CS7/20220214/fr-par/s3/aws4_request,
+        SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=527fb8e17949cdd083ba828e464d51737333db1cee8dca364509b553f4aeb8a7
+      User-Agent:
+      - aws-sdk-go/1.42.48 (go1.17.6; linux; amd64)
+      X-Amz-Content-Sha256:
+      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      X-Amz-Date:
+      - 20220214T140856Z
+    url: https://test-acc-scaleway-object-bucket-force-7443382734974225702.s3.fr-par.scw.cloud/
```

Comment:
Please use something like `https://test-acc-scaleway-object-bucket-force-7443382734974225702.s3.fr-par.scw.cloud/?versions=`
and response
```
<?xml version="1.0" encoding="UTF-8"?>\n<ListVersionsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Name>michael-versioning</Name><Prefix></Prefix><KeyMarker></KeyMarker><VersionIdMarker></VersionIdMarker><MaxKeys>1000</MaxKeys><EncodingType>url</EncodingType><IsTruncated>false</IsTruncated><DeleteMarker><Key>magic</Key><VersionId>uLkwWXneH5_gKvlqopQqKZaWhEUVY51m</VersionId><IsLatest>true</IsLatest><LastModified>2021-06-10T14:24:23.000Z</LastModified><Owner><ID>69a360b85b319b998292fbac13803cdc79898d1d2ed0f29f3a245874b0ebd97a</ID></Owner></DeleteMarker><Version><Key>magic</Key><VersionId>vrkAriN1FwpIibUSu9lH9VG65GYp28yB</VersionId><IsLatest>false</IsLatest><LastModified>2021-04-13T14:20:27.000Z</LastModified><ETag>&quot;d41d8cd98f00b204e9800998ecf8427e&quot;</ETag><Size>0</Size><Owner><ID>69a360b85b319b998292fbac13803cdc79898d1d2ed0f29f3a245874b0ebd97a</ID></Owner><StorageClass>STANDARD</StorageClass></Version></ListVersionsResult>
```